py/janitor/site/templates/cupboard/done-list.html

```diff
@@ -3,7 +3,7 @@
     {% include "cupboard/sidebar.html" %}
 {% endblock sidebar %}
 {% block page_title %}
-    Changes that have been merged or pushed
+    Cupboard - Done
```

Comment:
Please revert - this is less understandable than the previous description

Reply:
Done